src/services/state-manager/state-manager.service.ts

```diff
@@ -13,17 +16,19 @@ export class StateManagerService {
     return StateManagerService._instance
   }
 
-  public getBotState() {
+  public getBotState(): IState {
     return {
+      memberId: this._currentMemberId,
       state: this._currentBotState,
       step: this._currentBotStep,
       data: this._currentData
     }
   }
 
-  public setBotState(stateName: string, step: number, data?: string) {
+  public setBotState(memberId: string, stateName: StateNamesEnum, step: number, data?: string): void{
```

Comment:
Instead of listing all the arguments, why not simply have one parameter of type IState?

Reply:
Oh yes! I didn't think of that!